rust/rope/src/spans.rs

```diff
@@ -300,6 +300,23 @@ impl<T: Clone> Spans<T> {
         }
         *self = b.build();
     }
+
+    /// FIXME: Instead of iterating through all spans every time, another option would be to go
```

Comment:
Nittiest nit ever,  but I think the FIXME should not be part of the `/// doccomment`.

Reply:
whoops